README.md

```diff
@@ -41,7 +42,7 @@ Common use cases for ONNX Runtime:
 
 ## Data/Telemetry
 
-This project may collect usage data and send it to Microsoft to help improve our products and services. See the [privacy statement](docs/Privacy.md) for more details.
+Windows distributions of this project may collect usage data and send it to Microsoft to help improve our products and services. See the [privacy statement](docs/Privacy.md) for more details.
```

Comment:
Does the URL need to be full URL path or relative ones such `docs/Privacy.md`

Reply:
Relative path, as it's our ORT specific privacy statement page.